ambry-api/src/main/java/com.github.ambry/account/ContainerBuilder.java

```diff
@@ -23,16 +23,20 @@
  * This class is not thread safe.
  */
 public class ContainerBuilder {
+  // necessary
   private short id;
   private String name;
   private ContainerStatus status;
   private String description;
-  private boolean encrypted;
-  private final boolean previouslyEncrypted;
-  private boolean cacheable;
-  private boolean mediaScanDisabled;
   private short parentAccountId;
 
+  // optional
+  private boolean encrypted = false;
```

Comment:
The `Container` class has some default value constants that you could reference here.

Reply:
Done